recipes/pygadm/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pygadm" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygadm-{{ version }}.tar.gz
+  sha256: fa14ff5a134e2af2b9670da787c22e67a6cfb9c55552727a3f27c7ff355dad37
+
+build:
```

Comment:
Please also add the entry_points: https://github.com/12rambau/pygadm/blob/2c682d3781817eaa724a43120b720c35a45697b7/pyproject.toml#L67

Reply:
that's for developers only, let me remove it from the lib.

Reply:
removed in v0.2.0